=== modified file 'CMakeLists.txt'
--- old/CMakeLists.txt	2013-12-31 10:42:25 +0000
+++ new/CMakeLists.txt	2014-01-02 00:09:08 +0000
@@ -156,11 +156,15 @@
         set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${PIC_FLAG}" )
 
         # Thou shalt not link vaporware and tell us it's a valid DSO:
-        set( CMAKE_SHARED_LINKER_FLAGS "-Wl,--no-undefined" )
-        set( CMAKE_MODULE_LINKER_FLAGS "-Wl,--no-undefined" ) # needed by SWIG macros on linux
+        if(NOT APPLE)
+            set( CMAKE_SHARED_LINKER_FLAGS "-Wl,--no-undefined" )
+            set( CMAKE_MODULE_LINKER_FLAGS "-Wl,--no-undefined" ) # needed by SWIG macros on linux
+        endif()
 
         # Set default flags for Release build.
-        set( CMAKE_EXE_LINKER_FLAGS_RELEASE "-s" )
+        if(NOT APPLE)
+            set( CMAKE_EXE_LINKER_FLAGS_RELEASE "-s" )
+        endif()
 
     endif()
 

#=== modified file 'CMakeModules/download_boost.cmake'
#--- old/CMakeModules/download_boost.cmake	2013-12-31 16:58:05 +0000
#+++ new/CMakeModules/download_boost.cmake	2014-01-02 00:09:08 +0000
#@@ -43,7 +43,7 @@
# # tool_manager.cpp -> coroutine -> context (_jump_fcontext) (on OSX)
 
# set( BOOST_LIBS_BUILT
#-    context
#+    #context
#     #coroutine
#     date_time
#     #exception

